server/routes/conditions.ts

```diff
@@ -83,14 +83,12 @@ export default ({
       const bespokeConditions = record.licence?.licenceConditions?.bespoke || []
       const conditionsService = conditionsServiceFactory.forLicence(record)
       const conditions = conditionsService.getAdditionalConditions(record.licence)
-      let behaviours = conditionsService.getAbuseAndBehaviours(conditions) || []
```

Comment:
no longer needed as this is handled via config now as part of the `displayForEdit` functionality